### PR TITLE
Expense Status fix

### PR DESCRIPTION
**Pull request by asood123**
Variable name confusion was still causing bugs.

### CODE REVIEW

File: server/controllers/expenses.js [hunk 1]
```diff
@@ -225,15 +225,15 @@ export const pay = (req, res, next) => {
   }
 };
 
-function assertExpenseStatus(expense, status) {
-  if (status === status.PENDING && expense.status === status.APPROVED) {
+function assertExpenseStatus(expense, statusToConfirm) {
```
Comment (xdamman) [inline server/controllers/expenses.js]: What about `currentStatus`?

Comment (asood123) [inline server/controllers/expenses.js]: Thought about that, `currentStatus` is technically what `expense.status` should be. We want to check that the current status of an expense matches a particular status. Open to better wording

**Comment by asood123** [comment]
merging this for now to fix production

File: server/controllers/expenses.js (hunk 1 above)
Comment (xdamman) [inline server/controllers/expenses.js]: Then 'expectedStatus'?

Comment (asood123) [inline server/controllers/expenses.js]: that's better. Let's swap it next time one of us touches that code.

**Pull request merged**